### PR TITLE
Fixed form issues using toggleForm and header issues related to FigureInput

### PR DESCRIPTION
**Pull request by priyesh777**
Related to https://github.com/idmc-labs/Helix2.0/issues/105

### CODE REVIEW

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 1]
```diff
@@ -336,7 +336,7 @@ function FigureInput(props: FigureInputProps) {
             </Row>
             <Row>
                 <DateInput
-                    label="Start Date *"
+                    label={currentCatetory?.type === 'STOCK' ? 'Stock Date*' : 'Start Date'}
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:339]:
```suggestion
                    label={currentCatetory?.type === 'STOCK' ? 'Stock Date *' : 'Start Date *'}
```

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 2]
```diff
@@ -355,7 +355,7 @@ function FigureInput(props: FigureInputProps) {
                     options={dateAccuracyOptions}
                     keySelector={enumKeySelector}
                     labelSelector={enumLabelSelector}
-                    label="Start Date Accuracy"
+                    label={currentCatetory?.type === 'STOCK' ? 'Stock Date Accuracy*' : 'Start Date Accuracy'}
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:358]:
```suggestion
                    label={currentCatetory?.type === 'STOCK' ? 'Stock Date Accuracy' : 'Start Date Accuracy'}
```

File: src/views/Entry/EntryForm/FigureInput/index.tsx [hunk 3]
```diff
@@ -601,276 +602,280 @@ function FigureInput(props: FigureInputProps) {
...
-                                key={age.uuid}
-                                index={i}
-                                value={age}
-                                ageOptions={ageCategoryOptions}
-                                genderOptions={genderCategoryOptions}
-                                onChange={onAgeChange}
-                                onRemove={onAgeRemove}
-                                error={error?.fields?.disaggregationAgeJson?.members?.[age.uuid]}
+            {
+                value.isDisaggregated && (
```
Comment (tnagorra) [inline src/views/Entry/EntryForm/FigureInput/index.tsx:606]: Any reason the indentation was changed?

**Review by tnagorra — CHANGES_REQUESTED** [review]
The form library should also be removed

File: src/views/Entry/EntryForm/FigureInput/index.tsx (hunk 3 above)
Comment (priyesh777) [reply src/views/Entry/EntryForm/FigureInput/index.tsx:606]: I'm guessing the code editor modified when I used  ( Ctrl + Shift +i )

**Comment by priyesh777** [comment]
> The form library should also be removed

File: src/views/Entry/EntryForm/FigureInput/index.tsx (hunk 3 above)
Comment (tnagorra) [reply src/views/Entry/EntryForm/FigureInput/index.tsx:606]: Please revert this change!

**Review by AdityaKhatri — COMMENTED** [review]
@tnagorra , there are some loose typings left.

**Pull request merged**